week-1/E-strings/exercise.js

```diff
@@ -1,3 +1,5 @@
 // Start by creating a variable `message`
+var array = [1, 2, 3, 4];
```

Comment:
I'm not sure `array` is the best name for an array. Can you think of a more meaningful one?

Reply:
I`ve changed the array name as `arrayExample`.

Reply:
I have changed the array name as `arrayExample`.

Reply:
Hm how about something along the lines of `number`? The name of a variable should always describe its value (what's in it).

Reply:
`arrayNumbers` looks more related with the content of the array.